docs/labs/docker-plugin.rst

```diff
@@ -0,0 +1,147 @@
+=======================
+
+You can use the ``flocker-plugin-install`` tool which is part of the :ref:`installer <labs-installer>` to quickly install the Flocker Docker Plugin on a cluster you set up with that tool.
+
+Otherwise, if you want to install the Flocker Docker plugin manually, you can follow the following instructions.
+
+Manual Installation on Ubuntu 14.04
+===================================
+
+First install Flocker on some hosts.
+These instructions assume you have followed the :ref:`official Flocker install instructions <installflocker>`.
+See the :ref:`unofficial installer <labs-installer>` for a quicker method.
+
+On the same machine where you ran ``flocker-ca`` while installing Flocker, :ref:`generate a new API user certificate and key <generate-api>` for a user named ``plugin``.
+Upload these files to ``/etc/flocker/plugin.key`` and ``/etc/flocker/plugin.crt`` on the hosts where you want to run the Flocker Docker plugin.
+
+Then perform the following instructions on each of the hosts where you want to install the Flocker Docker plugin.
+
+Install Experimental Docker
+---------------------------
+
+Install the experimental build of Docker:
+
+.. prompt:: bash $
+
+    wget -qO- https://experimental.docker.com/ | sudo sh
+
+Install the Flocker Docker plugin
+---------------------------------
+
+On each of your container agent servers, install the Flocker plugin:
+
+.. prompt:: bash $
+
+    sudo apt-get install -y python-pip build-essential libssl-dev libffi-dev python-dev
+    sudo pip install git+https://github.com/clusterhq/flocker-docker-plugin.git
+
+Set up init scripts to run plugin on boot
+-----------------------------------------
+
+We need to define some configuration which will make it into the environment of the plugin:
+
+.. prompt:: bash $
+
+    FLOCKER_CONTROL_SERVICE_BASE_URL=https://your-control-service:4523/v1
+    MY_NETWORK_IDENTITY=1.2.3.4
+
+Replace ``your-control-service`` with the hostname of the control service you specified when you created your cluster.
+Replace ``1.2.3.4`` with the IP address of the host you are installing on (if your public and private IPs differ, it is generally best to use the *private* IP address of your hosts).
+
+Write out up an upstart script to automatically start the Flocker plugin on boot, including the configuration we just wrote out::
+
+    $ sudo su -
+    # cat <<EOF > /etc/init/flocker-docker-plugin.conf
+    # flocker-docker-plugin - flocker-docker-plugin job file
+    description "Flocker Plugin service"
+    author "ClusterHQ <support@clusterhq.com>"
+    respawn
+    env FLOCKER_CONTROL_SERVICE_BASE_URL=${FLOCKER_CONTROL_SERVICE_BASE_URL}
+    env MY_NETWORK_IDENTITY=${MY_NETWORK_IDENTITY}
+    exec flocker-docker-plugin
+    EOF
+    # service flocker-docker-plugin restart
+
+Now you should have the Flocker plugin running on this node, try running:
+
+.. prompt:: bash $
+
+    docker run -ti -v test:/data --volume-driver=flocker busybox sh
+
+On this node.
```

Comment:
does "On this node" follow on from the previous sentence? If so, I'd just say "try running the following command on this node"